frontend/components/help/HelpModal.tsx

```
@@ -0,0 +1,347 @@
+          </div>
+        </div>
+
+        {/* Article Body */}
+        <div className="flex-1 overflow-y-auto p-6">
+          <div className="prose prose-sm max-w-none">
+            <p className="text-gray-600 mb-6 text-base leading-relaxed">
+              {t(selectedArticle.descriptionKey)}
+            </p>
+            <div 
+              className="text-gray-700 leading-relaxed whitespace-pre-line"
+              dangerouslySetInnerHTML={{ __html: t(selectedArticle.contentKey) }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)